change/@ni-nimble-components-46e491a1-b49a-4d5c-b64e-6eb6fae50e0b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
fyi @Razvan1928 I was just reset on the PR but I have already put up new comments, I'm not sure if that was intentional